charts/topolvm/templates/mutatingwebhooks.yaml

```
@@ -21,6 +21,14 @@ webhooks:
       - key: {{ include "topolvm.pluginName" . }}/webhook
         operator: NotIn
         values: ["ignore"]
+      {{- with .Values.webhook.podMutatingWebhook.ignoreNamespaces }}
+      - key: kubernetes.io/metadata.name
+        operator: NotIn
+        values:
+          {{- range $v := .}}
```

nit: It might render correctly even without the space, but it's better to add it for consistency with the others.
```suggestion
          {{- range $v := . }}
```